frontend/src/metabase-types/guards/data-apps.ts

```diff
@@ -0,0 +1,17 @@
+import type {
+  ActionDashboardCard,
+  Dashboard,
+  DataAppNavItem,
+} from "metabase-types/api";
+
+export function isActionDashboardCard(
+  dashcard: unknown,
```

Comment:
From its uses. Seems like `dashcard` can either be an `ActionDashboardCard` or `DashboardOrderedCard`, so can we tighten the type here instead of using an `unknown`?